pkg/controller/versions/strategy_v1_1.go

```diff
@@ -80,6 +82,22 @@ func (v *versionStrategyV1_1) Validate(ctx context.Context, cl client.Client, sm
 			}
 		}
 	}
+	smmr := &v1.ServiceMeshMemberRoll{}
```

Comment:
Do we need this same check for the 1.0 and 1.2/2.0 validators?

Reply:
Yes, but I wanted to wait for your adjustment (re)introducing the loop over gateways